src/geo/projection/albers.js

```diff
@@ -58,5 +61,26 @@ export default {
         const lat = clamp(radToDeg(phi), -MAX_MERCATOR_LATITUDE, MAX_MERCATOR_LATITUDE);
 
         return new LngLat(lng, lat);
+    },
+
+    projectTilePoint(x: number, y: number): {x: number, y: number, z: number} {
```

Comment:
Just wondering: since mostly all projections apart from the globe implements these functions the same way, should we maybe move these to a superclass? (Maybe called as `FlatProjection` or something like that.)
I think it's fine to leave it now as-is, but it could be refactored later.

Reply:
Yes that's a good point, definitely something to consolidate here. I focused on making it work first, so those one-liners duplication are ok for now, but I'll keep a note in the main interface to improve that.